### PR TITLE
[PhpUnitBridge] remove deprecated features

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php [hunk 1]
```diff
@@ -16,7 +16,6 @@
 use PHPUnit\Framework\TestCase;
 use PHPUnit\Framework\TestSuite;
 use PHPUnit\Util\Blacklist;
-use PHPUnit\Util\Test;
```
Comment (Tobion) [inline src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php]: that seems to be unused also in earlier versions

Comment (xabbuh) [inline src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php]: see #22816

**Comment by nicolas-grekas** [comment]
changelog missing

**Comment by xabbuh** [comment]
changelog added (see #22834 and #22835 for related changes in older branches)

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**